gobblin-core/src/main/java/org/apache/gobblin/writer/PartitionedDataWriter.java

```diff
@@ -411,7 +415,7 @@ public void handleMessage(ControlMessage message) {
   /**
    * Get the serialized key to partitions info in {@link #state}
    */
-  private static String getPartitionsKey(int branchId) {
+  public static String getPartitionsKey(int branchId) {
```

Comment:
why changing to public ?

Reply:
This is needed in LinkedIn's internal publisher. Hence the change.